### PR TITLE
Connect ImpactView map

### LINKED ISSUE #5: Update carto table to reference impact view carto account
- Adjust data retrieval services as appropriate
- Update visualization zoom to point at dataset extent

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

Updates the app to use the map and data from the ImpactView organization.

Fallout left to address:
- Missing legend on map (#15)
- Fix search dropdown (#14)

## Demo

screen shot 2017-10-27 at 15 26 00

screen shot 2017-10-27 at 15 26 36

screen shot 2017-10-27 at 15 26 20

## Testing

If you already have a config.js file, copy your ACS key somewhere safe momentarily. Then, copy the new example config:
```
cp app/scripts/config.js.example app/scripts/config.js
```
Once done, re-add your ACS key to the new config.

The app should continue to behave as before, and all app features should work. Instead of museum data, you should see non-profit organizations in philadelphia on the map. The search feature is somewhat busted for now. If you use the search bar, don't geolocate (unless you're currently in Phila!) and limit non-organization searches to Phila Zip codes, e.g. 19123, 19107, etc.

Closes #5

### CODE REVIEW

File: app/scripts/views/museum/museum-partial.html [hunk 1]
```diff
@@ -23,24 +23,21 @@
...
                     <ul class="list-unstyled">
-                        <!-- TODO: Verify this value and make it more useful to user -->
-                        <li ng-if="museum.museum.discipl"><strong>Category:</strong> {{ museum.museum.discipl | imlsCategory }}</li>
-                        <!-- TODO: Filter to properly format phone number -->
-                        <li ng-if="museum.museum.phone"><strong>Phone:</strong> {{ ::museum.museum.phone | tel }}</li>
+                        <li ng-if="museum.museum.ntee_org_type_new"><strong>Category:</strong> {{ ::museum.museum.ntee_org_type_new }}</li>
+                        <li ng-if="museum.museum.org_size_label_new"><strong>Budget Size: </strong>{{ ::museum.museum.org_size_label_new }}</li>
+                        <li ng-if="museum.museum.year_founded_990"><strong>Year Founded: </strong>{{ ::museum.museum.year_founded_990 }}</li>
+                        <li ng-if="museum.museum.org_phone_new"><strong>Phone:</strong> {{ ::museum.museum.org_phone_new | tel }}</li>
                         <!-- TODO: Verify for bad data -->
```
Comment (KlaasH) [inline app/scripts/views/museum/museum-partial.html]: Does this TODO apply to ImpactView as well?

File: app/scripts/views/museum/tab-nearby-museums-directive.js [hunk 2]
```diff
@@ -14,23 +14,11 @@
             $scope.$watch(function () { return ctl.stateData; }, onStateDataChanged);
             $scope.$watch(function () { return ctl.areaData; }, onAreaDataChanged);
             $scope.$watch(function () { return ctl.isTabVisible; }, onTabVisibleChanged);
-            museumTypes = _.values(LegendMap);
         }
 
         // adds missing museum types and sorts alphabetically
```
Comment (KlaasH) [inline app/scripts/views/museum/tab-nearby-museums-directive.js]: Stale comment--now it just sorts.

**Review by KlaasH — APPROVED** [review]
Looks good. :+1:

Noticed a couple stale comments.  This sort of massive copy/modify operation will always create some confusion and ambiguity between what's still relevant and what's not, but comments that no longer apply are particularly tricky to interpret long after the fact, so we should try to modify or delete as we encounter them.

The pop-ups when you click on a location are also busted.  Is that on the radar?
image

**Comment by CloudNiner** [comment]
Added #17 for popup styling and removed stale comments.

**Pull request merged**